### DISCUSSION TITLE
Welcome to POE2loadingPainFix Discussions!

### DISCUSSION THREAD

**Discussion by CrimsonED1** [CrimsonED1/POE2loadingPainFix#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TheAPIguys** [comment]
Hello there,

quick question by default when i start the app version .8 it looks like this:
![image](https://github.com/user-attachments/assets/8c8c2a1f-7d9d-47b9-98e6-c7a197a4423c)

are all the core suppuse to be click on this section? It is not working correctly for me win11 amd 7 9700x

Cheers

**Reply by CrimsonED1** [reply]
This is The default selection for The in Limit affinity. 1 per Core workes best on my Setting. But you can change it. You can change it live while poe2 is Running. Add one by one. Until ist starting.
Is poe2 Not starting? (Not responding?)
Im also currently working for an Not responding fix.

**Reply by TheAPIguys** [reply]
i gets loading and windows doesn't crash but it was not finishing loading the map in poe2. I close the app and it would finish loading most cases. I will test some  more later and I can let you know what I found out. My general solution to my issues so far was deleting the shader folder in poe2 it seems to work for a bit but in long section I get those crashes would any app or affinity changes.

**Reply by CrimsonED1** [reply]
Hey, try the new version (0.9)

**Reply by TheAPIguys** [reply]
hey, thanks. I was using 0.8 ver for a couple days. It works but sometimes would freeze my poe2 and i have to toggle the always off and back to the auto to make the poe2 unfreeze. I try different settings nothing make those hard freeze stop. I will try with new version and let you know. Many thanks for your project man!

**Reply by CrimsonED1** [reply]
Yes, affinity was Not that good. The new mode (via Threads) shouldnt freeze poe. And Till now i got no hard freeze. But i only have my system… There are 2 trackbars  in expert mode, i havent documented yet. If you get a hard freeze you should set the pause higher.

and affinity, has the recovery mode now. Which will unfreeze poe. But it was making hardfreeze sometimes. So i wont recommenced. 1st try only the threaded mode

**Reply by TheAPIguys** [reply]
I am happy to report that I have a 3-4 hours section where i didn't experience any freezing or issues. I am still having an issue with the D3D12 Error "failed to create resource for texture []". If you have any experience with this let me know in a comment. I try vulkan but it get the same error message but with vulkan too. Cheers for all the help

**Reply by CrimsonED1** [reply]
Sounds good. I Never got this error. I have a NVIDIA 3060ti. And NVIDIA or amd GPU? So you have the newest driver?
Sounds Like a driver thing

edit: i use Vulkan.
